src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -396,6 +399,31 @@ AmendmentTableImpl::doValidation (
     return amendments;
 }
 
+std::vector <uint256>
+AmendmentTableImpl::getDesired ()
+{
```

Comment:
This looks very similar to `doValidation`.  Could the implementation of `getDesired` just call `doValidation` on an empty set?